### DISCUSSION TITLE
Cannot convert undefined to BigInt

### DISCUSSION THREAD

**Discussion by SDAdham** [mikro-orm/mikro-orm#6249] [General]
### Describe the bug

I have the same bug as https://github.com/mikro-orm/mikro-orm/issues/6092

### Reproduction

I have the same bug as https://github.com/mikro-orm/mikro-orm/issues/6092 Cannot convert undefined to a BigInt

### What driver are you using?

MySQL

### MikroORM version

6.4.0

### Node.js version

20.15.1

### Operating system

macos

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Discord](https://discord.gg/w8bjxFHS7X).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
> I have the same bug as https://github.com/mikro-orm/mikro-orm/issues/6092 Cannot convert undefined to a BigInt

That one is closed with a passing test case, pointing to a resolved issue with a passing test case is not a reproduction.

**Comment by SDAdham** [comment]
Hello @B4nan , I really apologize, I was meaning to close the issue long time ago but I got caught up. The issue is not within the mikro-orm.
